R-package/src/xgboost_R.cc

```diff
@@ -103,7 +103,7 @@ XGB_DLL SEXP XGDMatrixCreateFromMat_R(SEXP mat, SEXP missing) {
   }
```

Comment:
The for loop in lines 96-102 is still using multiple threads. We must revise the loop also.

Reply:
It's actually using the default.

> XGDMatrixCreateFromMat_omp_R, that takes in the extra parameter nthread

Maybe a new JSON based API.

Reply:
"The default" is set by `OMP_NUM_THREADS`, not `nthread.